.env

```diff
@@ -10,6 +10,8 @@ POSTGRES_USER=postgres
 REGISTRY_HOST=index.docker.io/
 REGISTRY_URI=uclalibrary
 SITE_URI=oralhistory-test.library.ucla.edu
+STAGING_HTTP_BASIC_USERNAME=oralhistory
```

Comment:
Goes with a this Brakeman Audit: https://github.com/scientist-softserv/oral-history/pull/33/files#r1144099295